HintList.py

```diff
@@ -57,15 +57,21 @@ def getHintGroup(group, world):
         if hint.name in world.always_hints and group == 'always':
             hint.type = 'always'
 
+        conditional_keep, type_append = True, False
```

Comment:
nit: prefer separate lines for assignment

Reply:
Fixed this